backend/e4h-services/ingestion-service/app/utils/im_service_client.py

```
@@ -1,12 +1,59 @@
-
-
-
-
-
-
-
-
+import json
+import requests
+
+from app.schemas.request_info import RequestInfo
```

Ruff (0.12.2)

`app.schemas.request_info.RequestInfo` imported but unused

Remove unused import: `app.schemas.request_info.RequestInfo`

(F401)